specification/kubernetesconfiguration/resource-manager/readme.ruby.md

```diff
@@ -12,17 +12,29 @@ azure-arm: true
 
 ``` yaml $(ruby) && $(multiapi)
 batch:
-  - tag: package-2019-11-01-preview'
-  - tag: package-2020-07-01-preview'
+  - tag: package-2021-03'
+  - tag: package-preview-2020-10
+  - tag: package-2020-07-01-preview
+  - tag: package-2019-11-01-preview
```

Comment:
Ruby SDK is retired. So you don't need to config for it.

Reply:
So do we remove it then?

Reply:
Removed ruby config